infrastructure/templates/template.json

```diff
@@ -1804,6 +1844,12 @@
           "name": "default/downloads",
```

Comment:
This should be set to:
"name": "default",
"type": "blobServices",
"apiVersion": "2019-04-01",